apps/wallet/src/ui/app/hooks/deepbook/useGetEstimate.ts

```diff
@@ -117,8 +114,9 @@ async function getPlaceMarketOrderTxn({
 
 	if (isAsk) {
 		const bigNumberBaseBalance = new BigNumber(baseBalance);
+		const oneSuiDeepBook = new BigNumber(1).shiftedBy(USDC_CONVERSION_RATE);
 
-		if (bigNumberBaseBalance.isLessThan(ONE_SUI_DEEPBOOK)) {
+		if (bigNumberBaseBalance.isLessThan(oneSuiDeepBook)) {
 			balanceToSwap = bigNumberBaseBalance.minus(
 				bigNumberBaseBalance.times(ESTIMATED_GAS_FEES_PERCENTAGE / 100),
 			);
```

Comment:
(I wanted to comment on line 127 but I can't) For wallet fee is `balanceToSwap` in deepbook denomination? Is there a case here that we apply fee on 1_000_000 which is (lets say) one SUI but this in network denomination should be 1 SUI = 1_000_000_000? so in the end we charge the wrong fee? (just want to confirm this is working as expected)

Reply:
@pchrysochoidis The wallet_fee is not in deepbook denomination, since we calculate it first and never call with `formatBalanceToLotSize`

Reply:
But that doesn't do anything about converting from network denomination to deepbook denomination. Maybe denomination is the same for both. and I misunderstood our discussion then.

How many mist is one SUI in deepbook? What about USDC?

Reply:
Ahh ok, I see the concern. The lot_size for SUI is the same as MIST in SUI for network, so it will work. We do not need to convert it. If in the future we do for other pools, then we will need to however. the USDC side we do not need to worry because it is also matching network value.

I will make a note in the document about this.

CC: @pchrysochoidis